project.clj

```diff
@@ -203,7 +201,7 @@
                                                ;; https://github.com/technomancy/leiningen/issues/2216
                                                ;; Hopefully we can remove those `nil`s (if we care)
                                                ;; and this comment when lein 2.7.2 is available.
-                                               ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
looks like there's a few whitespace oddities in this file (this and the two changes above)